### ISSUE TITLE
cursor style not working for terminado's demo: echo -e "\e[2 q"

### ISSUE THREAD

**Issue by hyperpallium** [xtermjs/xterm.js#1730]
I want to stop the cursor blinking.

Inspecting xtermjs source, it does seem to implement DECSCUSR `\e[# q` (# = [0-6], e.g. `echo -e "\e[2 q"` is a steady block) in `InputHandler.ts`, but it has no effect in the `single.py` demo. 
It starts with a blinking solid block, and none of the choices have any effect.

Oh, and I see where [suggested](https://github.com/xtermjs/xterm.js/issues/480) (BTW typo in the eg, missing space, should be `echo -e "\e[5 q"`), then [implemented](https://github.com/xtermjs/xterm.js/pull/492).

How to make `echo -e "\e[2 q"` work? (equivalent to `\033[2 q`, or `\x1b[2 q`), to stop the cursor blinking?
I wouldn't think it's because of terminado, but maybe it is? Maybe they use xtermjs before this, [from >1 year ago?

using latest (mobile) broswer: android firefox 62.0.3

**Comment by Tyriar** [comment]
Works for me on my macbook, must be terminado or your environment filtering it out? Also on mac you need to change `\e` to `\x1b`, maybe that's the problem?

Also, there is an API you can use which is the standard way for the terminal to set this:

```ts
term.setOption('cursorBlink', false);
```

**Comment by hyperpallium** [comment]
@Tyriar Thanks, looks like it's terminado. It works in my normal terminal - that would have the same environment, wouldn't it? I had tried all variants (`\e`, `\x1b`, `\033`).

Unfortunately, the API call also didn't work. I tried three different ways to access term, using terminado's demo as an example.

I also noticed that control keys don't work, which is a showstopper for my usages.

There are a few code layers here, and looks it would require careful study to get on top of it. It's a shame it doesn't "just work", but it is a pretty amazing concept, to have a full terminal in a browser.
